### PR TITLE
Added profile

### PR DESCRIPTION
**Pull request by nishith-shah-salesforce**

### CODE REVIEW

File: nishith.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+*fav adssg
```
Comment (mrmorris) [inline nishith.md]: Add your favorite meal

File: nishith.md [hunk 2]
```diff
@@ -1,2 +1,2 @@
-*fav adssg
-*dadfs sdf
+*fav meal: pizza
+*fav animal: lion
```
Comment (BhuvanaNa) [inline nishith.md:2]: The changes look good. Thanks

**Review by BhuvanaNa — APPROVED** [review]

**Review by mrmorris — APPROVED** [review]
Looks great now

**Comment by mrmorris** [comment]
👍

**Pull request merged**